CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.33.8] - 2022-05-10
```

Comment:
This needs a matching entry near the bottom of the file:

`[29.33.8]: https://github.com/linkedin/rest.li/compare/v29.33.7...v29.33.8`